modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/ObjectEntriesFDSPropsTransformer.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+interface ObjecEntriesDataRendererProps {
+	value: {
+		key: string;
+		name: string;
+		name_i18n: string;
+	}[];
+}
+
+export default function ObjectEntriesFDSPropsTransformer({
+	...otherProps
+}) {
+	return {
+		...otherProps,
+		customDataRenderers: {
+			objectEntrieDataRenderer: ({
+				value,
+			}: ObjecEntriesDataRendererProps) =>
+				value ? value.map((v) => v.name).join(', ') : ''
```

Comment:
@Artartur We should avoid abbreviations. Instead of `v`, please use `value`.

Reply:
Done!